src/containers/BooksForm.js

```diff
@@ -33,13 +34,24 @@ const BooksForm = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
-      <input name="title" onChange={handleChange} type="text" className="input" />
-      <select name="category" onChange={handleChange}>
-        {options}
-      </select>
-      <button type="submit" className="btn">Add Book</button>
-    </form>
+    <div className="bg-grey round-bottom box-shadow">
+      <div className="center max-width-90 border-top">
+        <div className="formTitle">Add New Book</div>
+        <form onSubmit={handleSubmit} className="bookForm">
+          <input
+            placeholder="Book Title"
+            name="title"
+            type="text"
+            value={title}
+            onChange={handleChange}
+          />
+          <select name="category" value={category} onChange={handleChange}>
+            {options}
+          </select>
```

Comment:
- [ ] <b>REQUIRED:</b> The title  and category you are using here are empty strings: ` ' ' ` checkout this [Video](https://www.loom.com/share/2ada06fc45a048bca8658b2b6a20e62a) and see whats the result is. So this is because you initialized them with an empty string on lines `18 and 19` and you were trying to give them the values in `handleChange` function which will re-render the component right after assigning value to those variables, and when the component re-renders, they will be initialized with empty strings again.  which means you can never be able to use those variables in that cycle.  however, there are many solutions to fix this.   

     -  Don't use any of those variables at all, instead get the values from book object(`book.title and book.category`)
     - Initialize those variables the value from book object. (`let title = book.title and let category = book.category`)